### PR TITLE
Expose background color and labelmap blend

### PR DESCRIPTION
**Pull request by scottwittenburg**
Adds a button to the top row which allows toggling between a set of pre-defined background colors.  The `publicAPI` already exposes a way to set the background color to an arbitrary value, but this PR makes sure that gets applied to the scene.  

This PR also exposes ability to set/get the `labelMapBlend`, a scalar value indicating how much we see the labelmap color map vs the other intensity components.  Higher values of `labelMapBlend` mean we see more of the labelmap colormap, while lower values mean we see more of the other intensity components.

This PR also emits new events when either of the above properties is changed.

### CODE REVIEW

File: src/UserInterface/Main/createBackgroundColorButton.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+function createBackgroundColorButton(store, mainUIRow) {
+  const viewerDOMId = store.id
+  const bgColorButton = document.createElement('div')
+  bgColorButton.innerHTML = `<input id="${viewerDOMId}-bgColorButton" type="checkbox" class="${style.toggleInput}" checked><label itk-vtk-tooltip itk-vtk-tooltip-top itk-vtk-tooltip-content="Toggle Background Color" class="${style.bgColorButton} ${style.toggleButton}" for="${viewerDOMId}-bgColorButton">${selectColorIcon}</label>`
+  const bgColorButtonLabel = bgColorButton.children[1]
+  applyContrastSensitiveStyle(store, 'invertibleButton', bgColorButtonLabel)
+  bgColorButton.addEventListener('change', event => {
+    event.preventDefault()
+    event.stopPropagation()
+    console.log('changed')
```
Comment (thewtex) [inline src/UserInterface/Main/createBackgroundColorButton.js:16]: rm

File: src/UserInterface/Main/createBackgroundColorButton.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    toggleBackgroundColor(store)
+  })
+  bgColorButton.addEventListener('click', event => {
+    event.preventDefault()
+    event.stopPropagation()
+    console.log('clicked')
```
Comment (thewtex) [inline src/UserInterface/Main/createBackgroundColorButton.js:22]: rm

File: src/UserInterface/toggleBackgroundColor.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+function toggleBackgroundColor(store) {
+  console.log('dang')
```
Comment (thewtex) [inline src/UserInterface/toggleBackgroundColor.js:2]: rm

**Comment by scottwittenburg** [comment]
> @scottwittenburg nice! A few debug statements noted inline.

Yes, I was just testing you.  You passed the test.

**Comment by thewtex** [comment]
Dang, good test :-D

**Pull request merged**